webknossos/webknossos/dataset/_utils/pims_tiff_reader.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+from os import PathLike
+from pathlib import Path
+from typing import Set, Tuple
+
+import numpy as np
+
+try:
+    from pims import FramesSequenceND
+except ImportError as e:
+    raise RuntimeError(
+        "Cannot import pims, please install it e.g. using 'webknossos[all]'"
+    ) from e
+
+try:
+    import tifffile
+except ImportError as e:
+    raise RuntimeError(
+        "Cannot import tifffile, please install it e.g. using 'webknossos[tifffile]'"
+    ) from e
```

Comment:
If both pims and tifffile are missing, it would be nice to just have one error message that tells users to install `webknossos[pims,tifffile]`.